miasm2/jitter/codegen.py

```diff
@@ -283,6 +283,8 @@ def traverse_expr_dst(self, expr, dst2index):
                         "0")
 
         else:
+            # Spot case like Cond?X:Y
+            assert -1 not in dst2index.values()
```

Comment:
I don't agree with this patch.

If you have a destination operation which looks like:
```
IRDst = ExprCond(cond, EAX, EBX)
```
The Generated code will be:
```
DST_case = cond?-1:-1;
DST_value = cond?EAX:EBX;
```
`DST_case` will be -1 in every cases (which is correct)
But the `DST_value` will be corerctly computed. So no problem in both cases.

Maybe I didn't get your point?